doc/book/hybrid.md

```diff
@@ -0,0 +1,119 @@
+# Encrypt and decrypt using hybrid cryptosystem
+
+Hybrid is an encryption mode that uses symmetric and public keys ciphers together.
+The idea is to take the advantages of the public key cryptography for sharing the
+keys and the speed of symmmetric encryption to encrypt the message.
```

Comment:
Rephrase to:

    The approach takes advantage of public key cryptography for sharing keys and symmetric encryption speed for encrypting messages.